### PR TITLE
U4X-667: Add directions to residence on patient registration

### PR DESCRIPTION
**Pull request by Daphne210**
The address where a specific individual can be found or contacted.

### CODE REVIEW

**Comment by jabahum** [comment]
fix conflicts

**Comment by Daphne210** [comment]
> fix conflicts

This has been fixed

File: packages/esm-ugandaemr-app/src/ugandaemr-configuration-overrrides.json [hunk 1]
```diff
@@ -653,13 +674,6 @@
                             ]
                         }
                     },
-                    "dispensing-dashboard-link": {
```
Comment (akileng56) [inline packages/esm-ugandaemr-app/src/ugandaemr-configuration-overrrides.json:656]: Does this remove the dispensing Menu? Or it was moved somewhere else

Comment (Daphne210) [reply packages/esm-ugandaemr-app/src/ugandaemr-configuration-overrrides.json]: > Does this remove the dispensing Menu? Or it was moved somewhere else

No it wasn't moved anywhere else. Must have been gotten mixed up when fixing conflicts. But I have fixed it back.

**Pull request merged**